docker/scripts/entrypoint.sh

```diff
@@ -84,6 +84,18 @@ while IFS='=' read -r -d '' var_name var_value; do
   fi
 done < <(env -0)
 
+if [ -n "${ZKV_CONF_BASE_PATH}" ]; then
+  BASE_CHAINS="${ZKV_CONF_BASE_PATH}/chains"
+
+  for chain in local testnet ; do
+    source_chain_dir="${BASE_CHAINS}/nh_${chain}";
+    dest_chain_dir="${BASE_CHAINS}/zkv_${chain}";
+    [ -d "$source_chain_dir" ] && [ ! -e "$dest_chain_dir" ] && \
+      echo "Move ${source_chain_dir} to ${dest_chain_dir}" && \
+      mv "${source_chain_dir}" "${dest_chain_dir}"
```

Comment:
Since we are changing this, should we use the `ls -s` command ?

Reply:
Mhm well I think there's no need, we just wanna make sure we have migrated/moved the data, not interested in which data actually.